P0009/P0009.bs

```diff
@@ -960,35 +952,32 @@ constexpr extents& operator=(const extents<OtherExtents...>& other);
 static constexpr size_t rank() const noexcept;
 ```
 
-* *Returns:* `sizeof...(Extents)`
+* *Returns:* `sizeof...(StaticExtents)`
 
 <br/>
 ```c++
 static constexpr size_t rank_dynamic() const noexcept;
 ```
 * *Returns:* 
-    * If `sizeof...(Extents)==0` is `true`, then zero;
-    * Otherwise, `((Extents==dynamic_extent)+...)`. 
-
-*[Note:* This is the number of dynamic extents. *—end note]*
+    * If `sizeof...(StaticExtents)==0` is `true`, then zero;
+    * Otherwise, `((StaticExtents==dynamic_extent)+...)`. 
```

Comment:
Need spaces around plus and double-equals.  See examples here for double-equals: http://eel.is/c++draft/input.output#streambuf.locales

Reply:
Need parens around the first fold expression.